scripts/compareResults.ts

```diff
@@ -0,0 +1,174 @@
+import stompit from "stompit"
+import generateMockPncQueryResult from "../tests/helpers/generateMockPncQueryResult"
+import MockPncGateway from "../tests/helpers/MockPncGateway"
+import CoreHandler from "../src/index"
+import differenceWith from "lodash.differencewith"
+import isEqual from "lodash.isEqual"
+import Pino from "pino"
+import type { Trigger } from "../src/types/Trigger"
+import type Exception from "../src/types/Exception"
+import type BichardResultType from "../src/types/BichardResultType"
+
+interface BichardResult {
+  incomingMessage: string
+  annotatedHearingOutcome: string
+  triggers?: Trigger[]
+  exceptions?: Exception[]
+}
+
+const logger: Pino.Logger = Pino({
```

Comment:
I think it would be nice if this was in it's own file (so it could be reused) and then imported here.

Reply:
Good point. It now lives under `src/utils/logging.ts` :)